src/RpcClient.ts

```diff
@@ -195,6 +193,36 @@ export class PostMessageRpcClient extends RpcClient {
     }
 }
 
+export class ReceiveOnlyPostMessageRpcClient extends PostMessageRpcClient {
```

Comment:
See discussion in https://github.com/nimiq/accounts/pull/106 on whether this class is actually necessary.